components/status/StatusActionsMore.vue

```diff
@@ -83,18 +84,12 @@ async function deleteStatus() {
 async function deleteAndRedraft() {
   if (await openConfirmDialog({
     title: t('confirm.delete_posts.title'),
+    description: t('confirm.delete_posts.description'),
     confirm: t('confirm.delete_posts.confirm'),
     cancel: t('confirm.delete_posts.cancel'),
   }) !== 'confirm')
     return
 
-  if (process.dev) {
-    // eslint-disable-next-line no-alert
-    const result = confirm('[DEV] Are you sure you want to delete and re-draft this post?')
-    if (!result)
-      return
-  }
-
```

Comment:
We will be asked by confirm dialog with this update so this duplicated alert won't be needed.

Reply:
Keep this confirm, it is for dev only

Reply:
@userquin All right, reverted this just now.